php-frontend/src/main/java/org/sonar/php/tree/impl/expression/ArrayInitializerFunctionTreeImpl.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * SonarQube PHP Plugin
+ * Copyright (C) 2010 SonarSource and Akram Ben Aissi
+ * sonarqube@googlegroups.com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+ */
+package org.sonar.php.tree.impl.expression;
+
+import com.google.common.collect.Iterators;
+import org.sonar.php.tree.impl.PHPTree;
+import org.sonar.php.tree.impl.SeparatedListImpl;
+import org.sonar.php.tree.impl.lexical.InternalSyntaxToken;
+import org.sonar.plugins.php.api.tree.Tree;
+import org.sonar.plugins.php.api.tree.expression.ArrayInitializerFunctionTree;
+import org.sonar.plugins.php.api.tree.expression.ArrayPairTree;
+import org.sonar.plugins.php.api.tree.lexical.SyntaxToken;
+import org.sonar.plugins.php.api.visitors.TreeVisitor;
+
+import java.util.Iterator;
+
+public class ArrayInitializerFunctionTreeImpl extends PHPTree implements ArrayInitializerFunctionTree {
+
+  private static final Kind KIND = Kind.ARRAY_INITIALIZER_FUNCTION;
+  private final InternalSyntaxToken arrayToken;
+  private final InternalSyntaxToken openParenthesis;
+  private final SeparatedListImpl<ArrayPairTree> arrayPairs;
+  private final InternalSyntaxToken closeParenthesis;
+
+  public ArrayInitializerFunctionTreeImpl(InternalSyntaxToken arrayToken, InternalSyntaxToken openParenthesis, SeparatedListImpl<ArrayPairTree> arrayPairs, InternalSyntaxToken closeParenthesis) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 195 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00103)